dashboard/app/controllers/gallery_activities_controller.rb

```diff
@@ -39,7 +39,7 @@ def create
       authorize! :save_to_gallery, @gallery_activity.user_level
 
       if @gallery_activity.save
-        render action: 'show', status: :created, location: @gallery_activity
+        return head :created
```

Comment:
Aha, it doesn't even look at the returned data (or display an error if the POST fails).